files/zh-cn/web/javascript/reference/global_objects/intl/numberformat/index.html

```diff
@@ -194,7 +194,7 @@ <h3 id="Using_notation">Using <code>notation</code></h3>
 // → 987,654E6
 
 console.log(new Intl.NumberFormat('zh-CN', { notation: "compact" }).format(987654321));
-// → 9.9亿
+// → 9.9 亿
```

Comment:
![image](https://user-images.githubusercontent.com/15844309/170852697-5d51a2f9-6154-48c9-9ab2-84088691bb59.png)

```suggestion
// → 9.9亿
```